systemd/tmpfiles-openqa.conf

```diff
@@ -1 +1,2 @@
 d /run/openqa 0755 _openqa-worker root
+d /var/lib/openqa/share 0755 _openqa-worker root
```

Comment:
But it is supposed to read from it. This is why I need to create the directory manually so the worker can be started.
The directory is created properly when installing openQA, because it isn't ghosted.

So then die solution is not tmpfiles.d but removing the need of using "share" dir when "cache" dir is configured, right?